pkg/helm/charts/kubepromstack/kubepromstack.go

```diff
@@ -0,0 +1,2430 @@
+// Package kubepromstack provides a helm chart for installing kube-prometheus-stack:
+// - https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack
+package kubepromstack
+
+import (
+	"bytes"
+	"text/template"
+	"time"
+
+	"github.com/oslokommune/okctl/pkg/helm"
+)
+
+// New returns an initialised helm chart:
+// - https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack
+func New(values *Values) *helm.Chart {
+	return &helm.Chart{
+		RepositoryName: "prometheus-community",
+		RepositoryURL:  "https://prometheus-community.github.io/helm-charts",
+		ReleaseName:    "kube-prometheus-stack",
+		Version:        "13.9.1",
+		Chart:          "kube-prometheus-stack",
+		Namespace:      "monitoring",
+		Timeout:        5 * time.Minute, // nolint: gomnd
```

Comment:
Maybe add a config.DefaultChartApplyTimeout instead of ignoring the gomnd. Would be neat if we have to adjust the timeout down the line

Reply:
We probably need to look at how we set up these charts, basically we should bring these options through to the caller. Introduced and fixed!